python/graphstorm/model/lp_gnn.py

```diff
@@ -94,6 +109,9 @@ def forward(self, blocks, pos_graph,
             # GNN message passing
             encode_embs = self.compute_embed_step(blocks, node_feats, input_nodes)
 
+        # Call emb normalization.
+        self.normalize_node_embs(encode_embs)
```

Comment:
what is this? why do you not get the returned embeddings?

Reply:
Fixed